src/platforms/javascript/common/lazy-load-sentry.mdx

```diff
@@ -1,8 +1,6 @@
 ---
 title: Lazy Loading Sentry
 sidebar_order: 11
-redirect_from:
-  - /platforms/javascript/common/
```

Comment:
https://docs.sentry.io/platforms/javascript/common/ should not redirect to https://docs.sentry.io/platforms/javascript/lazy-load-sentry/

Similar for the other redirects.